python-sdk/docs/astro/sql/operators/load_file.rst

```diff
@@ -221,6 +221,17 @@ Loading to MS SQL
 .. note::
    We do not support loading Unicode data to SQL Server due to limitations on the underlying ``pymssql`` library
 
+Loading to Duckdb
+~~~~~~~~~~~~~~~~~
+
+``load_file`` can load data to duckdb. If the database file is not specified in connection, it will be created
```

Comment:
```suggestion
``load_file`` can load data to duckdb. If the database file is not specified in the connection, it will assume the user is using an in-memory duckdb. For more information about the supported files, check the [official documentation](https://duckdb.org/docs/extensions/overview.html).
```

Reply:
https://github.com/astronomer/astro-sdk/pull/1695/commits/e09109bc9c1daa83b99b36608ed066ef95a2e3e4